package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.10.0",
   "description": "interface to mapbox services",
   "main": "lib/mapbox.js",
+  "bin": {
+    "mapbox-sdk": "./bin/mapbox-sdk"
```

Comment:
I know :smile: We actually have an internal command we've been working on called `mapbox` and we're trolling ourselves by not just swapping names (public: mapbox, internal: mbx or something else).